### PR TITLE
ffa: Export ffa.h to be used by host

### PR DESCRIPTION
**Pull request by jellesels-arm**
The ffa.h file is needed by the OP-TEE test suite. Export it so it can
be used by it. The corresponding test PR can be found [here](https://github.com/OP-TEE/optee_test/pull/614) 

Signed-off-by: Jelle Sels <jelle.sels@arm.com>

### CODE REVIEW

File: core/arch/arm/include/ffa.h [hunk 1]
```diff
@@ -9,7 +9,9 @@
 
 /* This is based on the FF-A 1.0 EAC specification */
 
+#if defined(ARM64) || defined(ARM32)
```
Comment (jforissier) [inline core/arch/arm/include/ffa.h:12]: Unrelated change?

Comment (jellesels-arm) [reply core/arch/arm/include/ffa.h]: No, it wasn't. However I changed it to be more clear.
For the spmc tests in xtest we need the ffa.h file, the ffa.h file needs the smccc.h. So I changed it to copy both files when the SPMC_TEST are selected.

**Review by jforissier — COMMENTED** [review]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Review by jenswi-linaro — COMMENTED** [review]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Review by etienne-lms — COMMENTED** [review]
Mention `CFG_SPMC_TESTS` in _mk/config.mk_?

**Comment by jellesels-arm** [comment]
> Mention `CFG_SPMC_TESTS` in _mk/config.mk_?

Ack

**Comment by jellesels-arm** [comment]
Any more comments on this? I would we good if we could merge the SPMC testing before the 3.19 release.
